pkg/webhook/webhook.go

```diff
@@ -159,6 +159,10 @@ func (w *Webhook) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
 	//Gogs needs to be checked before Github since it carries both Gogs and (incompatible) Github headers
 	case r.Header.Get("X-Gogs-Event") != "":
 		payload, err = w.gogs.Parse(r, gogs.PushEvent)
+	case r.Header.Get("X-Github-Event") == "ping":
+		rw.WriteHeader(http.StatusOK)
+		_, _ = rw.Write([]byte("Webhook received successfully"))
+		return
 	case r.Header.Get("X-GitHub-Event") != "":
 		payload, err = w.github.Parse(r, github.PushEvent)
```

Comment:
Having read more of this...

```go
        payload, err = w.github.Parse(r, github.PushEvent, github.PingEvent)
        if _, ok := payload.(github.PingPayload); ok {
            _, _ = rw.Write([]byte("Webhook received successfully"))
            return
        }
```
You don't need the `rw.WriteHeader(http.StatusOK)` as that's the default.